src/fftvis/core/antenna_gridding.py

```diff
@@ -142,62 +118,43 @@ def check_antpos_griddability(
     max_denominator: int = 10**6,
     max_factor: int = 1000,
 ) -> Tuple[bool, Dict[Any, np.ndarray], np.ndarray]:
-    """
-    Check if the antenna positions can be gridded. This function checks if the
-    antenna positions can be scaled to integers using a linear transformation.
-    It first infers the 2D lattice basis from the antenna positions, then checks
-    if the positions can be scaled to integers. If they can, it returns the
-    modified antenna positions and the transformation matrix. If not, it returns
-    the original antenna positions and an identity transformation matrix.
+    """Check if antenna positions can be gridded.
```

Comment:
Again, let's not remove the entire description. Have a short summary sentence, then move the body of hte description to a new paragraph.